stage1/init/init.go

```diff
@@ -669,7 +669,7 @@ func stage1() int {
 	unifiedCgroup, err := cgroup.IsCgroupUnified("/")
 	if err != nil {
 		log.FatalE("error determining cgroup version", err)
```

Comment:
`log.FatalE` ends up calling the stdlibs `log.Fatal` which does an `os.Exit(1)`.

Unless I'm wrong, all the instances of `log.FatalE` followed by `return ...` are redundant returns.